content/en/docs/instrumentation/erlang/getting-started.md

```diff
@@ -267,3 +264,118 @@ iex(2)>
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Exporting to the OpenTelemetry Collector
+
+The [Collector](/docs/collector/) provides a vendor
+agnostic way to receive, process and export telemetry data. The package
+[opentelemetry_exporter](https://hex.pm/packages/opentelemetry_exporter)
+provides support for both exporting over both HTTP (the default) and GRPC to the
+collector which can then export Spans to a self-hosted service like Zipkin or
+Jaeger, as well as commercial services. See the [OpenTelemetry Collector's
+contrib
+repo](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter)
+for a full list of available exporters in the Collector.
+
+For testing purposes the `opentelemetry-erlang` repo has a Collector
+configuration,
+[config/otel-collector-config.yaml](https://github.com/open-telemetry/opentelemetry-erlang/blob/main/config/otel-collector-config.yaml)
+that can be used as a starting point. This configuration is used in
+[docker-compose.yml](https://github.com/open-telemetry/opentelemetry-erlang/blob/main/docker-compose.yml)
+to start the Collector with receivers for both HTTP and GRPC that then export to
+Zipkin also run by [docker-compose](https://docs.docker.com/compose/).
+
+To export to the running Collector the `opentelemetry_exporter` package must be
+added to the project's dependencies:
+
+{{< tabs Erlang Elixir >}}
+
+{{< tab >}}
+{deps, [{opentelemetry_api, "~> 1.0"}, 
+        {opentelemetry, "~> 1.0"},
+        {opentelemetry_exporter, "~> 1.0"}]}.
+{{< /tab >}}
+
+{{< tab >}}
+def deps do
+  [
+    {:opentelemetry_api, "~> 1.0"},
+    {:opentelemetry, "~> 1.0"},
+    {:opentelemetry_exporter, "~> 1.0"}
+  ]
+end
+{{< /tab >}}
+
+{{< /tabs >}}
+
+It should then be added to the configuration of the Release, it should be before the SDK
+Application to ensure the exporter's dependencies are started before the SDK
+attempts to initialize and use the exporter.
+
+Example of Release configuration in `rebar.config` and for [mix's Release
+task](https://hexdocs.pm/mix/Mix.Tasks.Release.html):
+
+{{< tabs Erlang Elixir >}}
+
+{{< tab >}}
+%% rebar.config
+{relx, [{release, {my_instrumented_release, "0.1.0"},
+         [opentelemetry_exporter,
+	      {opentelemetry, temporary},
+          my_instrumented_app]},
+
+       ...]}.
+{{< /tab >}}
+
+{{< tab >}}
+# mix.exs
```

Comment:
Do want two leading `#`s here?

Reply:
Na, Elixir convention is just one `#`, while Erlang convention for comments on a line of its own is two `%%`.

Reply:
Ok, I was just wondering about the inconsistency. `##` is used on these lines (of this page):

- L133, at https://github.com/open-telemetry/opentelemetry.io/pull/1167/files#diff-bfa43494e057aca10a8d8fd612bd226d5c546f3b9aaffeb551a62fdf43ed7f22R133
- L368, at https://github.com/open-telemetry/opentelemetry.io/pull/1167/files#diff-bfa43494e057aca10a8d8fd612bd226d5c546f3b9aaffeb551a62fdf43ed7f22R368

Should those two instance be using a single `#` too?

Reply:
Ah! Yea, those should be a single `#` I'll push another commit fixing that up.